docs/api.rst

```diff
@@ -47,15 +47,14 @@ Unix file descriptors, where you have ``dup()`` and ``close()``, and Windows'
 Handles vs ``PyObject *``
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. XXX I don't like this sentence, but I can't come up with anything better
-   right now. Please rephrase/rewrite :)
-
-In the old Python/C API, multiple ``PyObject *`` references to the same object
-are completely equivalent to each other. Therefore they can be passed to Python/C
-API functions interchangeably. As a result, ``Py_INCREF`` an ``Py_DECREF`` can
-be called with any reference to an object as long as the total number of calls
-of `incref` is equal to the number of calls of `decref` at the end of the object
-lifetime.
+In order to fully understand the way HPy handles work, it is useful to discuss
+the Python/C API ``Pyobject *`` pointer. In the latter, these pointers always
```

Comment:
```suggestion
the Python/C API ``Pyobject *`` pointer. These pointers always
```
There is no "latter" to reference since there is only a single case (i.e. the pointer).